Assets/Scripts/Jump.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Jump : MonoBehaviour {
+	
+	public float jumpSpeed = 8.0f;
+
+	private Vector3 moveDirection = Vector3.zero;
+	private bool grounded = true;
```

Comment:
Since this is a strongly-typed language let's preface all booleans with "is". Should this be`isOnGround`?

Reply:
Since it's private, we could use the convention of prepending "_" to it. This would allow us to have `_isOnGround` here, and `isOnGround` for the public boolean below.